### DISCUSSION TITLE
Add https://github.com/paralab/SymPyGR to list in wiki?

### DISCUSSION THREAD

**Discussion by cottrell** [sympy/sympy#21950] [General]
Might be good to include this here: https://github.com/sympy/sympy/wiki/Code-Generation-Notes somewhere

**Comment by moorepants** [comment]
Anyone can edit the wiki if you have github account (I think).

**Comment by asmeurer** [comment]
You can add it if you want. I should note though that that page was only ever intended to be my own notes on code generation, so it may be out of date.
